### PR TITLE
Water Wave Cup

### PR DESCRIPTION
**Pull request by satyasonu**
I have created a cup which is filled with water and creating a wave. Please consider this as my submission. Here I have used simple HTML and CSS in it.
The below is page snapshot. Please check once.
image

### CODE REVIEW

File: entries/cupWaterWave.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        height: 78vh;
+        text-align: center;
+        /* background-color: #262626; */
+        border-bottom-left-radius: 2em;
+        border-bottom-right-radius: 2em;
+        border-top-left-radius: 0.1em;
+        border-top-right-radius: 0.1em;
+        /* padding-top: -50px; */
+        /* transform: translate(-50%, -50%); */
+        background: url(https://image.ibb.co/fmHm66/wave.png);
```
Comment (Metroxe) [inline entries/cupWaterWave.html:29]: You cannot use an external image/file, if you want to use an image you will need to embed the image as a base64 inline.

**Review by Metroxe — CHANGES_REQUESTED** [review]

**Comment by satyasonu** [comment]
I have changed the external image link into **base64** inline. Please check my code.

File: entries/cupWaterWave.html (hunk 1 above)
Comment (satyasonu) [reply entries/cupWaterWave.html]: Thanks for the suggestion. I have changed external image file to base64 inline.

**Review by Metroxe — APPROVED** [review]

**Comment by Metroxe** [comment]
all good, thank you!

**Pull request merged**